csharp/src/Google.Protobuf.Test/UnknownFieldSetTest.cs

```diff
@@ -138,6 +138,29 @@ public void TestClone(IMessage message)
             Assert.AreEqual(message.ToByteArray(), otherEmptyMessage.ToByteArray());
         }
 
+        [Test]
+        public void TestClone_LengthDelimited()
+        {
+            var destUnknownFieldSet = new UnknownFieldSet();
```

Comment:
I feel that currently the test is hard to read. How about:

```
            var unknownVarintField = new UnknownField();
            unknownVarintField.AddVarint(99);
            
            var unknownLengthDelimitedField1 = new UnknownField();
            unknownLengthDelimitedField1.AddLengthDelimited(ByteString.CopyFromUtf8("some data"));

            var unknownLengthDelimitedField2 = new UnknownField();
            unknownLengthDelimitedField2.AddLengthDelimited(ByteString.CopyFromUtf8("some more data"));

            var destUnknownFieldSet = new UnknownFieldSet();
            destUnknownFieldSet.AddOrReplaceField(997, unknownVarintField);
            destUnknownFieldSet.AddOrReplaceField(999, unknownLengthDelimitedField1);
            destUnknownFieldSet.AddOrReplaceField(999, unknownLengthDelimitedField2);  // replace the field
```

Reply:
done